salalib/importutils.cpp

```diff
@@ -103,7 +102,10 @@ namespace depthmapX {
                     catch (Communicator::CancelledException) {
                         return 0;
                     }
-                    catch (pexception) {
+                    catch (std::invalid_argument&) {
```

Comment:
Maybe replace these multiple catch statements with `catch(std::logic_error&)` to catch anything thrown out of the STL?